src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Flex, Heading, HeadingProps, HStack, Link, LinkProps, List, ListItem } from '@chakra-ui/react'
+
+const FooterHeading = (props: HeadingProps) => (
+	<Heading size='sm' {...props} />
+)
+
+const FooterLink = (props: LinkProps) => (
+	<Link size='sm' {...props} />
```

Comment:
Not sure why do you need these wrappers?
Just use Link and Heading.

Reply:
They're not necessary, but otherwise the repetition seemed bad. To me this was like the equivalent of writing CSS to make all the headings and links in the footer some certain style.

Reply:
If it's bad I can just do more straightforward way.

Reply:
I think in this case it's preferable to stay with the following method:

```js
const styles = {
    size: 'sm',
}

{...styles}
```

Also you might want to drop hstack and use only Flex and maybe Box to make achieve better responsiveness, eventually, I can get that done.

Reply:
Oh, `{...styles}` is pretty nice, thanks. I made another attempt with feedback incorporated.